docs/vector/vector-intro.md

```diff
@@ -1,7 +1,7 @@
 # geotrellis.vector
 
->"Raster is faster but vector is correcter."
-— Somebody
+> “Yes raster is faster, but raster is vaster and vector just SEEMS more corrector.”
```

Comment:
I see :)
While reading both docs, I thought somebody forgot to copy the quotation from raster-intro to vector-intro.